app/controllers/stripe_active_cards_controller.rb

```diff
@@ -11,16 +11,14 @@ def create
       Rails.logger.info("Stripe Add New Card Success - #{current_user.username}")
```

Comment:
I noticed there are no DD `tags` for the `destroy` action

Reply:
Correct - the goal of this PR is to add monitors needed to transition existing monitors on Timber over to Datadog. The `destroy` action was not included in the Timber monitors. If we want to monitor other actions that would be another PR.